shared/contract-utils/test/checkpointing.js

```diff
@@ -96,27 +96,4 @@ contract('Checkpointing', () => {
 
     await assertRevert(checkpointing.addCheckpoint(time - 1, value))
   })
-
-  const UINT64_OVERFLOW = (new web3.BigNumber(2)).pow(new web3.BigNumber(64))
-  const UINT192_OVERFLOW = (new web3.BigNumber(2)).pow(new web3.BigNumber(192))
-
-  it('fails if set value is too high', async () => {
```

Comment:
In line with https://github.com/aragonone/voting-connectors/pull/37#pullrequestreview-337786992, we should also make sure to double check in both the TokenWrapper and VotingAggregator that we're catching the casts correctly for any input values.